### PR TITLE
astrometry-net: fix image2pnm and other python scripts.

### PR DESCRIPTION
**Pull request by dstndstn**
- do not replace the shebang; the Makefile already does that based on the PYTHON_SCRIPT envvar.
- PYTHON_SCRIPT *must* be set to the virtualenv that is created.
- Add test of the 'image2pnm' script that uses this.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/astrometry-net.rb [hunk 1]
```diff
@@ -57,6 +55,7 @@ def install
   test do
     xy = Language::Python.major_minor_version Formula["python@3.8"].opt_bin/"python3"
     ENV["PYTHONPATH"] = libexec/"lib/python#{xy}/site-packages"
```
Comment (Bo98) [inline Formula/astrometry-net.rb:57]: Maybe an idea to remove these two lines so that the test picks up the error.

**Review by Bo98 — APPROVED** [review]

File: Formula/astrometry-net.rb (hunk 1 above)
Comment (dstndstn) [reply Formula/astrometry-net.rb]: Thanks, I hadn't noticed that (despite being surprised that the tests were passing... ugh).  I removed them in the next commit.

**Comment by dstndstn** [comment]
ps this is for issue #54256

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@dstndstn Thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**